slices/languageSlice.ts

```diff
@@ -13,14 +13,22 @@ const initialState: LanguageProficiencyState = {
 
 // actions
 
-export const addLanguagePro = (test: any) => ({
+export const addLanguagePro = (language: languageInterface) => ({
```

Comment:
I personally prefer Hungarian notations when it comes to interfaces, i.e. ILanguage